src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/custom.py

```diff
@@ -4,6 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 import base64
+from six.moves import configparser
```

Comment:
[CI is complaining](https://travis-ci.org/Azure/azure-cli/jobs/524569053#L1570) that this import statement should be moved below the `urlsplit` import.